### DISCUSSION TITLE
Save settings should auto-populate name with something useful

### DISCUSSION THREAD

**Discussion by PatrickF1** [rxhanson/Rectangle#486] [General]
Maybe something like `Rectangle_settings.json`. I've seen other apps with similar JSON backup settings do this. It'll be a lot better than untitled.json, which could lead to a user saving their settings and not remembering later what it's for and deleting it.

**Comment by PatrickF1** [comment]
Oops sorry this should have been a discussion because it's not a bug. Sorry!

**Reply by rxhanson** [reply]
(Not a problem, it's just a button for me to convert it)

**Comment by rxhanson** [comment]
Good point. I settled on `RectangleConfig.json`. This will go into the next release.
